index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>abdulsomodtribute page</title>
+    <link rel="stylesheet" href="stylesheet.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="main-head">Robot technology</h1>
+      <p id="title">Robotics reserch what is use for and more information</p>
+      <figure id="img-div">
+        <img id="image" src="robot.jpg" alt="Robotics information" />
+        <figcaption id="img-caption">
+          humanoid robot this is the first robot that was maid and it is White in color
+        </figcaption>
+      </figure>
+      <section id="tribute-info">
+        <h3>All this fast equipment save time an our energy</h3>
+        <ul>
+          <li>
+            robot, any automatically operated machine that replaces human effort, though it may not
+            resemble human beings in appearance or perform functions in a humanlike manner. By
+            extension, robotics is the engineering discipline dealing with the design, construction,
+            and operation of robots.
+          </li>
+          <li>
+            The concept of artificial humans predates recorded history (see automaton), but the
+            modern term robot derives from the Czech word robota (“forced labour” or “serf”), used
+            in Karel Čapek’s play R.U.R. (1920). The play’s robots were manufactured humans,
+            heartlessly exploited by factory owners until they revolted and ultimately destroyed
+            humanity. Whether they were biological, like the monster in Mary Shelley’s Frankenstein
+            (1818), or mechanical was not specified, but the mechanical alternative inspired
```

Comment:
Can we make the list no less than 15 list items? This helps to highlight the design and make it look more like the sample webpage.

I think we should add a year to each list and make the year styling a bit different. See screenshot below:

![image](https://user-images.githubusercontent.com/36057474/147682482-3677bea3-6798-4f34-838f-8afdb10c35d2.png)